arrow-core/src/main/kotlin/arrow/core/extensions/setk.kt

```diff
@@ -92,15 +118,21 @@ interface SetKMonoidK : MonoidK<ForSetK> {
     fix().setCombineK(y)
 }
 
-@extension
+@Deprecated(
+  "Typeclass instance have been moved to the companion object of the typeclass.",
+  ReplaceWith("Hash.setK<A>()", "arrow.core.extensions", "arrow.core.extensions"),
+  DeprecationLevel.WARNING)
```

Comment:
```suggestion
  ReplaceWith("Hash.set<A>()", "arrow.core.set", "arrow.typeclasses.Hash"),
```

You need to use fully qualified packages in the `ReplaceWith` statement. `SetK` is getting removed so we're not going to add an extension on `Show` for it.